clar2wasm/src/words/arithmetic.rs

```diff
@@ -397,4 +363,41 @@ mod tests {
             evaluate("(ok true)"),
         );
     }
+
+    #[test]
+    fn test_regress_three() {
+        crosscheck(
+            "(* 0 5 -34028236692093846346337460743176821146)",
+            Ok(Some(Value::Int(0))),
+        );
+    }
+
+    #[test]
+    fn test_regress_sub() {
+        crosscheck("(- 5 398 3 4 5)", Ok(Some(Value::Int(-405))));
+    }
+
+    #[test]
+    fn test_regress_sub_unary() {
+        crosscheck("(- 5)", Ok(Some(Value::Int(-5))));
+    }
+
```

Comment:
Could you also add a check for `uint` here?

```rust
    #[test]
    fn test_regress_sub_unary_uint() {
        crosscheck("(- u5)", Err(()));
    }
```

Reply:
Done